.github/test-matrix.json

```diff
@@ -0,0 +1,20 @@
+{
+  "otp_version": ["24.0.4", "23.3.4.2", "22.3.4.20"],
+  "elixir_version": ["1.12.2", "1.11.4", "1.10.4"],
+  "include": [
+    {
+      "otp_version": "21.3.8.24",
+      "elixir_version": "1.10.4",
+      "rebar3_version": "3.15.2"
+    }
```

Comment:
I was trying (unsuccessfully) to get a solution working where we can pull in a config and assign it to the strategy. I know it can be done (there are examples) but I just aborted it temporarily to get things at least working. Until we can get that working, we have to copy the matrix around. Otherwise, it's going to be a maintenance headache.

The `include` part of a matrix strategy means "also add this one combination to the final matrix". In our case, 1.10 isn't compatible with OTP >22 but we do support Elixir 1.10. Basically, just creating a matrix that covers min to max based on https://hexdocs.pm/elixir/1.12/compatibility-and-deprecations.html#compatibility-between-elixir-and-erlang-otp. Conversely, you can tell it to drop some specific set from the matrix with the `exclude`

I do think the exclude can be dropped and is leftover from one of the other projects.